sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -237,7 +237,7 @@ case class CreateDataSourceTableAsSelectCommand(
       className = table.provider.get,
       partitionColumns = table.partitionColumnNames,
       bucketSpec = table.bucketSpec,
-      options = table.storage.properties ++ pathOption,
+      options = table.properties ++ table.storage.properties ++ pathOption,
```

Comment:
I think the PR title is a bit misleading. We do respect TBLPROPERTIES in CTAS, but we do not include them as DS v1 write options, which should be specified by OPTIONS, or SERDEPROPERTIES.